libraries/cfg-core/lib/graph/ControlFlowGraph.ts

```diff
@@ -91,7 +103,7 @@ export class ControlFlowGraph<S> {
    */
   private getIncomingEdgesMap(): Map<string, Edge[]> {
     const map = new Map<string, Edge[]>();
-    for (const edge of this._edges) {
+    for (const edge of this.edges) {
```

Comment:
Why do we use the public getter and not the internal structure?

Reply:
No clue, fixed it!